src/Nethermind/Nethermind.Core/Crypto/KeccakHash.cs

```diff
@@ -621,5 +637,127 @@ public static void ReturnState(ref ulong[] state)
                 state = Array.Empty<ulong>();
             }
         }
+
+        public static void KeccakF1600Avx512F(Span<ulong> state)
```

Comment:
SkipLocalsInit?

Reply:
Added, but it doesn't change anything as the whole method is in registers